test/integration/sites/forums_controller_test.rb

```diff
@@ -11,10 +11,6 @@ def setup
   end
 
   def test_edit
-    Account.any_instance.expects(:provider_can_use?).returns(true).at_least_once
```

Comment:
Why is this removed here?
I mean, I know forums are deprecated and are going to be deleted soon, but why this specific test?... :thinking:

Reply:
Because it's the only one breaking CI 🙂.